model/src/main/scala/explore/model/encoders.scala

```diff
@@ -12,8 +12,10 @@ import lucuma.core.math.Declination
 import lucuma.core.math.RightAscension
 
 import ModelOptics._
+import sttp.model.Uri
 
 object encoders {
+  implicit val uriEncoder: Encoder[Uri] = Encoder.encodeString.contramap[Uri](_.toString)
```

Comment:
Do we need tests?

Reply:
I'll look into it. Do you know if we have a `Gen[Uri]` ?

Reply:
Maybe scalacheck includes one?

Reply:
This `Uri` is from `sttp-model` and I can't find `Arbitrary` or `Eq` instances for it.

We can look into implement circe's `CodecTests` for everything that we `Encode`/`Decode` in another PR, and implement a proper `Gen[Uri]` in that context.